app/assets/stylesheets/moderators.scss

```diff
@@ -163,13 +163,12 @@
 }
 
 .mod-actions-menu-btn {
-  transition: 0.3s all ease;
```

Comment:
Is this transition property removed on purpose?

Reply:
yeap: it wasn't really doing anything useful (or anything at all) (unless i missed something) and it was only causing a little visual jump on mobile.